prereise/gather/solardata/nsrdb/sam.py

```diff
@@ -86,13 +84,12 @@ def retrieve_data(solar_plant, email, api_key, year="2016", rate_limit=0.5):
                 }
             )
             data_site["ts_id"] = range(1, len(data_site) + 1)
-            data_site["plant_id"] = i[0]
+            data_site["plant_id"] = i
 
             power = 0
             for j, axis in enumerate([0, 2, 4]):
                 pv_dict = {
-                    # capacity in KW (DC)
-                    "system_capacity": i[1] * 1000.0 * ilr,
```

Comment:
We remove the scaling of MW to kW here so that we can remove the scaling of kW to MW later, since it's a linear model?

Reply:
Exactly. I followed the documentation before without knowing it was linear. I had to test it for this feature and realized it was not necessary to scale before feeding the model